charts/iap/values.yaml

```diff
@@ -15,9 +15,11 @@
 iap:
   image:
     repository: quay.io/oauth2-proxy/oauth2-proxy
-    tag: v6.0.0
+    tag: v6.1.1
     pullPolicy: IfNotPresent
 
+  # custom_provider_ca_secret: provider-ca-secret 
```

Comment:
Please add a comment line to explain what this does.

Reply:
added comment, thx for hint